a10_octavia/controller/worker/tasks/server_tasks.py

```diff
@@ -288,7 +275,7 @@ def execute(self, member, nat_flavor=None, nat_pool=None):
         return
 
 
-class MemberReleaseSubnetAddr(MemberBaseTask):
+class MemberReleaseSubnetAddr(BaseNetworkTask):
```

Comment:
It can be shifted to `ReleaseSubnetAddressForMember`

Reply:
done